chapters/fr/chapter1/3.mdx

```diff
@@ -96,13 +96,15 @@ from transformers import pipeline
 
 classifier = pipeline("zero-shot-classification")
 classifier(
-    "This is a course about the Transformers library",  # C'est un cours sur la bibliothèque Transformers
+    "This is a course about the Transformers library",
+    # C'est un cours sur la bibliothèque Transformers

```

Comment:
This is a really cool example to show readers what the text is in their native tongue - nice idea!

Since French has reasonable coverage model-wise, one idea for the future would be to change the code so that we use French models instead of English ones. This will require some work on our side to make sure the Colab notebooks are generated per language, but could be interesting to explore in the future

Reply:
This is something I had in mind. I'll submit a list of ideas to you on Discord.